ApiBundle/Transformer/ContentCollectionTransformer.php

```diff
@@ -30,7 +31,7 @@ public function transform($contentCollection, $contentType = null)
             array()
         ));
 
-        if ($contentType) {
+        if ($contentType && $this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_CREATE_CONTENT_TYPE_FOR_CONTENT)) {
```

Comment:
I'm not sure you check the correct right: I thought the issue was on the content creation. The ROLE_ACCESS_CREATE_CONTENT_TYPE_FOR_CONTENT should be used to allow the creation of content type, not content